portfolio/src/main/java/com/google/sps/comments/VoteCommentServlet.java

```diff
@@ -31,25 +33,28 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Retrieve commentId from the request
-    long commentId = Long.parseLong(request.getParameter("commentId"));
-    boolean typeOfVote = Boolean.parseBoolean(request.getParameter("vote"));
-    
-    //If key dows not exists, catch the error
-    try{
-      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-      Key commentEntityKey = KeyFactory.createKey("Comment", commentId);
-      Entity comment = datastore.get(commentEntityKey);
+    if (AuthServlet.isRegistered()) {
+      // Retrieve commentId from the request
+      long commentId = Long.parseLong(request.getParameter("commentId"));
+      boolean typeOfVote = Boolean.parseBoolean(request.getParameter("vote"));
+
+      //If key dows not exists, catch the error
```

Comment:
```suggestion
      // If key does not exist, catch the error.
```
Please use spaces at the beginning of comments